### ISSUE TITLE
TinyMCE inside a metabox creates console errors in GB

### ISSUE THREAD

**Issue by kmgalanakis** [WordPress/gutenberg#6305]
I use this sample code to create a metabox in the editor 

```
add_action( 'add_meta_boxes',  function() {
	add_meta_box(
		'my_custom_tinymce_metabox',
		'My metabox title',
		'my_metabox_callback_function'
	);
});

function my_metabox_callback_function( $post ) {
	wp_editor(
		htmlspecialchars_decode( get_post_meta( $post->ID, 'metabox_tinymce_meta_key', true ) ),
		'metabox_tinymce_id',
		$settings = array(
			'textarea_name' => 'metabox_tinymce_name',
		)
	);
}

add_action( 'save_post',
	function( $post_id ) {
		if ( ! empty( $_POST['metabox_tinymce_name'] ) ) {
			$data = htmlspecialchars( $_POST['metabox_tinymce_name'] );
			update_post_meta( $post_id, 'metabox_tinymce_meta_key', $data );
		}
	}
);
```

The code works fine in the classic editor but in Gutenberg it creates a few console errors

```
Uncaught SyntaxError: Unexpected token <
tinymce.min.js?ver=4607-20180123:13 Uncaught TypeError: b is not a constructor
    at n (tinymce.min.js?ver=4607-20180123:13)
    at Object.r [as init] (tinymce.min.js?ver=4607-20180123:13)
    at q.<anonymous> (tinymce.min.js?ver=4607-20180123:13)
    at Array.<anonymous> (tinymce.min.js?ver=4607-20180123:3)
    at b (tinymce.min.js?ver=4607-20180123:1)
    at r (tinymce.min.js?ver=4607-20180123:3)
    at tinymce.min.js?ver=4607-20180123:3
    at HTMLScriptElement.f (tinymce.min.js?ver=4607-20180123:3)
```

Am I doing something wrong?

I see this comment https://github.com/WordPress/gutenberg/issues/5181#issuecomment-375599883 but how could I use the same version as the version used in GB also in my metabox?

**Comment by youknowriad** [comment]
Pretty sure this is a duplicate, but I'll have hard time retrieving the original issue ;) 

Once we merge it will be fixed, this is caused by conflicting TinyMCE versions (see #3840 )

**Comment by kmgalanakis** [comment]
It's always really hard for me to search for tickets everytime I need to open a new one, so I totally get you :)

If there is another ticket for this, feel free to close this as duplicate.

Thanks

**Comment by danielbachhuber** [comment]
I've run into this same issue too.

**Comment by youknowriad** [comment]
closed by #6848
